pages/tokens/applying-tokens.en.mdx

```diff
@@ -3,15 +3,17 @@
 There are two ways how you can apply tokens to your selection:
 
 ### Default behaviour (left-click)
-When you left-click a token this token is being put on your selection (multiple layers selected are possible!). For certain tokens we assume defaults, such as for `Colors` we assume you want to apply `Fill`.
+When you left-click on a token, it is applied to your current selection (multiple layers can be selected). For certain tokens, we provide default assumptions. For example, with `Colors`, we assume that you want to apply the token as a `Fill`.
 
 ### Specifying what to apply (right-click)
-You can right-click tokens to specify what property should be set, such as in `Spacing` the `Left`, `Top`, `Right`, `Bottom` or `Gap` properties individually
+You can also right-click on tokens to specify a particular token property you would like to set such as `Left`, `Top`, `Right`, `Bottom`, or `Gap` properties in the case of Spacing; can also be set individually.  
+<img src='/token-property.png' alt="Select token properties - step 1" width="800"/>
 
 ### Control where to apply changes
-You can control what parts of your design get updated by utilizing the Apply selector in the bottom left of the plugin. You can choose between
-- Apply to page (currently selected page will get updated)
-- Apply to document (all pages get updated, possibly slow)
-- Apply to selection (only the currently selected nodes will get updated)
+You can control the scale of which your designs are updated by utilising the `Apply selector` located at the bottom right of the plugin. You have the following options to choose from:
```

Comment:
I think 'the scale of which' is confusing. What scale? Scale in design to me usually means what resolution as in 2x, 1x etc